### ISSUE TITLE
Incorrect handling of tags containing `/` char

### ISSUE THREAD

**Issue by awilliams** [drone-plugins/drone-github-release#24]
If a tag is used that contains `/` (for example `test/v1.2.3`), then the plugin will link the release to the tag `v1.2.3`.

I believe this happens because the `filepath.Base` function is used to extract the tag name from the commit reference. [Source](https://github.com/drone-plugins/drone-github-release/blob/master/plugin.go#L119)

**Comment by tboerger** [comment]
@bradrydzewski should we only strip the prefix?

**Comment by bradrydzewski** [comment]
Can we use `DRONE_TAG` which should already strip the rpefix?

```
params["DRONE_TAG"] = strings.TrimPrefix(m.Curr.Commit.Ref, "refs/tags/")
```

**Comment by tboerger** [comment]
> Can we use DRONE_TAG which should already strip the rpefix?

Sounds good to me. Maybe @awilliams can/wants to create a pull request for that?

**Comment by awilliams** [comment]
Thanks @bradrydzewski @tboerger, that sounds like a good solution. I'll create a PR for it.

**Comment by tboerger** [comment]
Fixed by #26
